3.processing_features/scripts/1.sc_cosmicqc.py

```diff
@@ -109,216 +120,298 @@ def reshape_data(df: pd.DataFrame, feature_col: str, feature_name: str) -> pd.Da
     "Plate_3": "Plate A",
     "Plate_3_prime": "Plate B",
     "Plate_5": "Plate C",
-    "Plate_6": "Plate D"
+    "Plate_6": "Plate D",
 }
 
 # Add the plate_alias column
-combined_df['plate_alias'] = combined_df['Image_Metadata_Plate'].map(plate_alias_mapping)
+combined_df["plate_alias"] = combined_df["Image_Metadata_Plate"].map(
+    plate_alias_mapping
+)
 
 # Print output
 print(combined_df.shape)
 combined_df.head()
 
 
+# In[5]:
+
+
+from PIL import Image
+
+def do_not_adjust_image_brightness(image: Image.Image) -> Image.Image:
```

Comment:
It's confusing why this function exists? Consider adding to docstring. Also, imports and functions prob should be defined earlier

Reply:
Ah fair, I missed this, I will add that context! :D